src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(Color color, double radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public void draw() {
+        StringBuilder stringBuilder = new StringBuilder();
+        stringBuilder.append("Figure: circle, ")
+                        .append("area: ").append(getArea()).append(" sq.units, ")
+                        .append("radius: ").append(radius).append(" units, ")
+                        .append("color: ").append(color.name());
```

Comment:
Ensure that the `color` variable is accessible in this context. If `color` is a field in the `Figure` class, it should be accessible here. Otherwise, you may need to adjust the access or initialization of `color`.